src/index.html

```diff
@@ -12,7 +12,72 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <section class="stars stars--0">
+      <p class="stars__star"></p>
```

Comment:
The `p` tag should be indented with 2 spaces relative to its parent `section` tag. Also, there should not be a blank line between the opening `section` tag and its first child `p` tag.